examples/print_affinity.cpp

```diff
@@ -53,22 +53,22 @@ int main(int argc, char** argv)
     for (int i = 0; i < omp_get_num_threads(); ++i) {
         int cpu = sched_getcpu();
 #pragma omp ordered
-{
-        if (i) {
-            thread_list << ",";
+        {
+            if (i) {
+                thread_list << ",";
+            }
+            thread_list << cpu;
         }
-        thread_list << cpu;
-}
```

Comment:
At least the `#pragma` is in the right spot now.  The curly braces are erroneously indented imo